components/utils.py

```diff
@@ -133,13 +133,13 @@ def deploy_ssl_certificate(private_or_public, destination, group, cert):
     else:
         ctx.abort_operation("Certificates may only be 'private' or 'public', "
                             "not {0}".format(private_or_public))
-    ctx.logger.info(
+    ctx.logger.debug(
         "Deploying {0} SSL certificate in {1} for group {2}".format(
             private_or_public, destination, group))
     sudo_write_to_file(cert, destination)
-    ctx.logger.info("Setting permissions ({0}) and ownership ({1}) of SSL "
-                    "certificate at {2}".format(
-                        permissions, ownership, destination))
+    ctx.logger.debug("Setting permissions ({0}) and ownership ({1}) of "
+                     "SSL certificate at {2}".format(
+                         permissions, ownership, destination))
```

Comment:
1) Why double quotes? Is it because of the parens around the formatting blocks?
2) other times you've aligned .format with the indent...also line 171 below.